examples/data/e2e_tests/setup_scripts/import_all.sh

```diff
@@ -61,8 +61,8 @@ curl -v \
     -D $HEADERS_FILE \
     "$URL/api/login?username=$USER&password=$PASSWORD"
 
-import_function $SCHEMA_DIR "schema"
-import_function $MAPPING_TABLE_DIR "mappingTable"
+#import_function $SCHEMA_DIR "schema"
+#import_function $MAPPING_TABLE_DIR "mappingTable"
```

Comment:
How come these imports are no longer needed? It would seem that they must be present for the dataset to work properly...